src/afl-fuzz.c

```diff
@@ -762,22 +762,22 @@ int main(int argc, char **argv_orig, char **envp) {
 
   }
 
-  if (get_afl_env("AFL_NO_FORKSRV")) afl->no_forkserver = 1;
-  if (get_afl_env("AFL_NO_CPU_RED")) afl->no_cpu_meter_red = 1;
-  if (get_afl_env("AFL_NO_ARITH")) afl->no_arith = 1;
-  if (get_afl_env("AFL_SHUFFLE_QUEUE")) afl->shuffle_queue = 1;
-  if (get_afl_env("AFL_FAST_CAL")) afl->fast_cal = 1;
+  if (afl->afl_env.afl_no_forksrv) afl->no_forkserver = 1;
```

Comment:
I think all of these are redundant.
Of they are only read here onc, we might not need a second .env variable 🤔 what's everybody's opinion on this?

Reply:
That is true. Then, we'd have to add flags for it, right? Or, can we remove the variables from afl_state and keep them in env_struct?

Reply:
Well in these cases (`no_arith`, `shuffle_queue`, `fast_cal`, `no_forkserver`) there are already some flags.
I'd suggest we leave those out of the env struct for now, then decide what to do for the others later.